Can not  set up the required third party libraries after the docker build

Python 3.9.5
Debian  BullEyes 5.10.179-2.
gcc (Ubuntu 10.3.0-1ubuntu1) 10.3.0

Can not  set up the required third party libraries after the docker build successfully

I install Miniconda for python and also try installing libpng-dev  libjpeg-dev libz3-dev
Here is the (long) error message after launching ``` bash configure.sh reinstall```. I cut some parts because log was too long...

````
-- Installing: /code/SuperBuild/install/share/opencv4/lbpcascades/lbpcascade_silverware.xml
[ 59%] Completed 'opencv'
[ 59%] Built target opencv
In file included from Src/Ply.h:56,
                 from Src/VertexStream.h:33,
                 from Src/FEMTree.h:51,
                 from Src/PoissonRecon.cpp:47:
Src/CoredMesh.h: In constructor ‘CoredFileMeshData<Index, VertexFactory>::CoredFileMeshData(const VertexFactory&, const char*) [with Index = int; VertexFactory = VertexFactory::Factory<float, VertexFactory::PositionFactory<float, 3>, VertexFactory::NormalFactory<float, 3>, VertexFactory::ValueFactory<float>, VertexFactory::EmptyFactory<float> >]’:
Src/CoredMesh.h:156:67: warning: ‘XXXXXX’ directive writing 6 bytes into a region of size between 1 and 1024 [-Wformat-overflow=]
  156 |    if( fileHeader && strlen(fileHeader) ) sprintf( _fileName , "%sXXXXXX" , fileHeader );
      |                                                                   ^~~~~~
In file included from /usr/include/stdio.h:866,
                 from Src/PoissonRecon.cpp:40:
/usr/include/x86_64-linux-gnu/bits/stdio2.h:38:34: note: ‘__builtin___sprintf_chk’ output between 7 and 1030 bytes into a destination of size 1024
   38 |   return __builtin___sprintf_chk (__s, __USE_FORTIFY_LEVEL - 1,
      |          ~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   39 |       __glibc_objsize (__s), __fmt,
      |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   40 |       __va_arg_pack ());
      |       ~~~~~~~~~~~~~~~~~
In file included from Src/Ply.h:56,
                 from Src/VertexStream.h:33,
                 from Src/FEMTree.h:51,
                 from Src/PoissonRecon.cpp:47:
Src/CoredMesh.h: In constructor ‘CoredFileMeshData<Index, VertexFactory>::CoredFileMeshData(const VertexFactory&, const char*) [with Index = int; VertexFactory = VertexFactory::Factory<float, VertexFactory::PositionFactory<float, 3>, VertexFactory::DynamicFactory<float> >]’:
Src/CoredMesh.h:156:67: warning: ‘XXXXXX’ directive writing 6 bytes into a region of size between 1 and 1024 [-Wformat-overflow=]
  156 |    if( fileHeader && strlen(fileHeader) ) sprintf( _fileName , "%sXXXXXX" , fileHeader );
      |                                                                   ^~~~~~
In file included from /usr/include/stdio.h:866,
                 from Src/PoissonRecon.cpp:40:
/usr/include/x86_64-linux-gnu/bits/stdio2.h:38:34: note: ‘__builtin___sprintf_chk’ output between 7 and 1030 bytes into a destination of size 1024
   38 |   return __builtin___sprintf_chk (__s, __USE_FORTIFY_LEVEL - 1,
      |          ~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   39 |       __glibc_objsize (__s), __fmt,
      |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   40 |       __va_arg_pack ());
      |       ~~~~~~~~~~~~~~~~~
In file included from Src/Ply.h:56,
                 from Src/VertexStream.h:33,
                 from Src/FEMTree.h:51,
                 from Src/PoissonRecon.cpp:47:
Src/CoredMesh.h: In constructor ‘CoredFileMeshData<Index, VertexFactory>::CoredFileMeshData(const VertexFactory&, const char*) [with Index = int; VertexFactory = VertexFactory::Factory<float, VertexFactory::PositionFactory<float, 3>, VertexFactory::NormalFactory<float, 3>, VertexFactory::DynamicFactory<float> >]’:
Src/CoredMesh.h:156:67: warning: ‘XXXXXX’ directive writing 6 bytes into a region of size between 1 and 1024 [-Wformat-overflow=]
  156 |    if( fileHeader && strlen(fileHeader) ) sprintf( _fileName , "%sXXXXXX" , fileHeader );
      |                                                                   ^~~~~~
In file included from /usr/include/stdio.h:866,
                 from Src/PoissonRecon.cpp:40:
/usr/include/x86_64-linux-gnu/bits/stdio2.h:38:34: note: ‘__builtin___sprintf_chk’ output between 7 and 1030 bytes into a destination of size 1024
   38 |   return __builtin___sprintf_chk (__s, __USE_FORTIFY_LEVEL - 1,
      |          ~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   39 |       __glibc_objsize (__s), __fmt,
      |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   40 |       __va_arg_pack ());
      |       ~~~~~~~~~~~~~~~~~
In file included from Src/Ply.h:56,
                 from Src/VertexStream.h:33,
                 from Src/FEMTree.h:51,
                 from Src/PoissonRecon.cpp:47:
Src/CoredMesh.h: In constructor ‘CoredFileMeshData<Index, VertexFactory>::CoredFileMeshData(const VertexFactory&, const char*) [with Index = int; VertexFactory = VertexFactory::Factory<float, VertexFactory::PositionFactory<float, 3>, VertexFactory::ValueFactory<float>, VertexFactory::DynamicFactory<float> >]’:
Src/CoredMesh.h:156:67: warning: ‘XXXXXX’ directive writing 6 bytes into a region of size between 1 and 1024 [-Wformat-overflow=]
  156 |    if( fileHeader && strlen(fileHeader) ) sprintf( _fileName , "%sXXXXXX" , fileHeader );
      |                                                                   ^~~~~~
In file included from /usr/include/stdio.h:866,
                 from Src/PoissonRecon.cpp:40:
/usr/include/x86_64-linux-gnu/bits/stdio2.h:38:34: note: ‘__builtin___sprintf_chk’ output between 7 and 1030 bytes into a destination of size 1024
   38 |   return __builtin___sprintf_chk (__s, __USE_FORTIFY_LEVEL - 1,
      |          ~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   39 |       __glibc_objsize (__s), __fmt,
      |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   40 |       __va_arg_pack ());
      |       ~~~~~~~~~~~~~~~~~
In file included from Src/Ply.h:56,
                 from Src/VertexStream.h:33,
                 from Src/FEMTree.h:51,
                 from Src/PoissonRecon.cpp:47:
Src/CoredMesh.h: In constructor ‘CoredFileMeshData<Index, VertexFactory>::CoredFileMeshData(const VertexFactory&, const char*) [with Index = int; VertexFactory = VertexFactory::Factory<float, VertexFactory::PositionFactory<float, 3>, VertexFactory::RGBColorFactory<float> >]’:
Src/CoredMesh.h:156:67: warning: ‘XXXXXX’ directive writing 6 bytes into a region of size between 1 and 1024 [-Wformat-overflow=]
  156 |    if( fileHeader && strlen(fileHeader) ) sprintf( _fileName , "%sXXXXXX" , fileHeader );
      |                                                                   ^~~~~~
In file included from /usr/include/stdio.h:866,
                 from Src/PoissonRecon.cpp:40:
/usr/include/x86_64-linux-gnu/bits/stdio2.h:38:34: note: ‘__builtin___sprintf_chk’ output between 7 and 1030 bytes into a destination of size 1024
   38 |   return __builtin___sprintf_chk (__s, __USE_FORTIFY_LEVEL - 1,
      |          ~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   39 |       __glibc_objsize (__s), __fmt,
      |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   40 |       __va_arg_pack ());
      |       ~~~~~~~~~~~~~~~~~
In file included from Src/Ply.h:56,
                 from Src/VertexStream.h:33,
                 from Src/FEMTree.h:51,
                 from Src/PoissonRecon.cpp:47:
Src/CoredMesh.h: In constructor ‘CoredFileMeshData<Index, VertexFactory>::CoredFileMeshData(const VertexFactory&, const char*) [with Index = int; VertexFactory = VertexFactory::Factory<float, VertexFactory::PositionFactory<float, 3>, VertexFactory::NormalFactory<float, 3>, VertexFactory::ValueFactory<float>, VertexFactory::DynamicFactory<float> >]’:
Src/CoredMesh.h:156:67: warning: ‘XXXXXX’ directive writing 6 bytes into a region of size between 1 and 1024 [-Wformat-overflow=]
  156 |    if( fileHeader && strlen(fileHeader) ) sprintf( _fileName , "%sXXXXXX" , fileHeader );
      |                                                                   ^~~~~~
In file included from /usr/include/stdio.h:866,
                 from Src/PoissonRecon.cpp:40:
/usr/include/x86_64-linux-gnu/bits/stdio2.h:38:34: note: ‘__builtin___sprintf_chk’ output between 7 and 1030 bytes into a destination of size 1024
   38 |   return __builtin___sprintf_chk (__s, __USE_FORTIFY_LEVEL - 1,
      |          ~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   39 |       __glibc_objsize (__s), __fmt,
      |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   40 |       __va_arg_pack ());
      |       ~~~~~~~~~~~~~~~~~
In file included from Src/Ply.h:56,
                 from Src/VertexStream.h:33,
                 from Src/FEMTree.h:51,
                 from Src/PoissonRecon.cpp:47:
Src/CoredMesh.h: In constructor ‘CoredFileMeshData<Index, VertexFactory>::CoredFileMeshData(const VertexFactory&, const char*) [with Index = int; VertexFactory = VertexFactory::Factory<float, VertexFactory::PositionFactory<float, 3>, VertexFactory::EmptyFactory<float> >]’:
Src/CoredMesh.h:156:67: warning: ‘XXXXXX’ directive writing 6 bytes into a region of size between 1 and 1024 [-Wformat-overflow=]
  156 |    if( fileHeader && strlen(fileHeader) ) sprintf( _fileName , "%sXXXXXX" , fileHeader );
      |                                                                   ^~~~~~
In file included from /usr/include/stdio.h:866,
                 from Src/PoissonRecon.cpp:40:
/usr/include/x86_64-linux-gnu/bits/stdio2.h:38:34: note: ‘__builtin___sprintf_chk’ output between 7 and 1030 bytes into a destination of size 1024
   38 |   return __builtin___sprintf_chk (__s, __USE_FORTIFY_LEVEL - 1,
      |          ~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   39 |       __glibc_objsize (__s), __fmt,
      |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   40 |       __va_arg_pack ());
      |       ~~~~~~~~~~~~~~~~~
In file included from Src/Ply.h:56,
                 from Src/VertexStream.h:33,
                 from Src/FEMTree.h:51,
                 from Src/Poisson

......................
......................
......................
......................
                 from Src/PoissonRecon.cpp:40:
/usr/include/x86_64-linux-gnu/bits/stdio2.h:38:34: note: ‘__builtin___sprintf_chk’ output between 4 and 1027 bytes into a destination of size 1024
   38 |   return __builtin___sprintf_chk (__s, __USE_FORTIFY_LEVEL - 1,
      |          ~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   39 |       __glibc_objsize (__s), __fmt,
      |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   40 |       __va_arg_pack ());
      |       ~~~~~~~~~~~~~~~~~
Src/PoissonRecon.cpp:376:84: warning: ‘PR_’ directive writing 3 bytes into a region of size between 0 and 1023 [-Wformat-overflow=]
  376 |   else                                                  sprintf( tempHeader , "%s%cPR_" , tempPath , FileSeparator );
      |                                                                                    ^~~
In file included from /usr/include/stdio.h:866,
                 from Src/PoissonRecon.cpp:40:
/usr/include/x86_64-linux-gnu/bits/stdio2.h:38:34: note: ‘__builtin___sprintf_chk’ output between 5 and 1028 bytes into a destination of size 1024
   38 |   return __builtin___sprintf_chk (__s, __USE_FORTIFY_LEVEL - 1,
      |          ~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   39 |       __glibc_objsize (__s), __fmt,
      |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   40 |       __va_arg_pack ());
      |       ~~~~~~~~~~~~~~~~~
Src/PoissonRecon.cpp:375:82: warning: ‘PR_’ directive writing 3 bytes into a region of size between 1 and 1024 [-Wformat-overflow=]
  375 |   if( tempPath[ strlen( tempPath )-1 ]==FileSeparator ) sprintf( tempHeader , "%sPR_" , tempPath );
      |                                                                                  ^~~
In file included from /usr/include/stdio.h:866,
                 from Src/PoissonRecon.cpp:40:
/usr/include/x86_64-linux-gnu/bits/stdio2.h:38:34: note: ‘__builtin___sprintf_chk’ output between 4 and 1027 bytes into a destination of size 1024
   38 |   return __builtin___sprintf_chk (__s, __USE_FORTIFY_LEVEL - 1,
      |          ~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   39 |       __glibc_objsize (__s), __fmt,
      |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   40 |       __va_arg_pack ());
      |       ~~~~~~~~~~~~~~~~~
Src/PoissonRecon.cpp:376:84: warning: ‘PR_’ directive writing 3 bytes into a region of size between 0 and 1023 [-Wformat-overflow=]
  376 |   else                                                  sprintf( tempHeader , "%s%cPR_" , tempPath , FileSeparator );
      |                                                                                    ^~~
In file included from /usr/include/stdio.h:866,
                 from Src/PoissonRecon.cpp:40:
/usr/include/x86_64-linux-gnu/bits/stdio2.h:38:34: note: ‘__builtin___sprintf_chk’ output between 5 and 1028 bytes into a destination of size 1024
   38 |   return __builtin___sprintf_chk (__s, __USE_FORTIFY_LEVEL - 1,
      |          ~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   39 |       __glibc_objsize (__s), __fmt,
      |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   40 |       __va_arg_pack ());
      |       ~~~~~~~~~~~~~~~~~
Src/PoissonRecon.cpp:375:82: warning: ‘PR_’ directive writing 3 bytes into a region of size between 1 and 1024 [-Wformat-overflow=]
  375 |   if( tempPath[ strlen( tempPath )-1 ]==FileSeparator ) sprintf( tempHeader , "%sPR_" , tempPath );
      |                                                                                  ^~~
In file included from /usr/include/stdio.h:866,
                 from Src/PoissonRecon.cpp:40:
/usr/include/x86_64-linux-gnu/bits/stdio2.h:38:34: note: ‘__builtin___sprintf_chk’ output between 4 and 1027 bytes into a destination of size 1024
   38 |   return __builtin___sprintf_chk (__s, __USE_FORTIFY_LEVEL - 1,
      |          ~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   39 |       __glibc_objsize (__s), __fmt,
      |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   40 |       __va_arg_pack ());
      |       ~~~~~~~~~~~~~~~~~
Src/PoissonRecon.cpp: In function ‘void Execute(UIntPack<CSignatures ...>, const AuxDataFactory&) [with Real = float; AuxDataFactory = VertexFactory::EmptyFactory<float>; unsigned int ...FEMSigs = {3, 3, 3}]’:
Src/PoissonRecon.cpp:376:84: warning: ‘PR_’ directive writing 3 bytes into a region of size between 0 and 1023 [-Wformat-overflow=]
  376 |   else                                                  sprintf( tempHeader , "%s%cPR_" , tempPath , FileSeparator );
      |                                                                                    ^~~
In file included from /usr/include/stdio.h:866,
                 from Src/PoissonRecon.cpp:40:
/usr/include/x86_64-linux-gnu/bits/stdio2.h:38:34: note: ‘__builtin___sprintf_chk’ output between 5 and 1028 bytes into a destination of size 1024
   38 |   return __builtin___sprintf_chk (__s, __USE_FORTIFY_LEVEL - 1,
      |          ~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   39 |       __glibc_objsize (__s), __fmt,
      |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   40 |       __va_arg_pack ());
      |       ~~~~~~~~~~~~~~~~~
Src/PoissonRecon.cpp:375:82: warning: ‘PR_’ directive writing 3 bytes into a region of size between 1 and 1024 [-Wformat-overflow=]
  375 |   if( tempPath[ strlen( tempPath )-1 ]==FileSeparator ) sprintf( tempHeader , "%sPR_" , tempPath );
      |                                                                                  ^~~
In file included from /usr/include/stdio.h:866,
                 from Src/PoissonRecon.cpp:40:
/usr/include/x86_64-linux-gnu/bits/stdio2.h:38:34: note: ‘__builtin___sprintf_chk’ output between 4 and 1027 bytes into a destination of size 1024
   38 |   return __builtin___sprintf_chk (__s, __USE_FORTIFY_LEVEL - 1,
      |          ~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   39 |       __glibc_objsize (__s), __fmt,
      |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   40 |       __va_arg_pack ());
      |       ~~~~~~~~~~~~~~~~~
Src/PoissonRecon.cpp:376:84: warning: ‘PR_’ directive writing 3 bytes into a region of size between 0 and 1023 [-Wformat-overflow=]
  376 |   else                                                  sprintf( tempHeader , "%s%cPR_" , tempPath , FileSeparator );
      |                                                                                    ^~~
In file included from /usr/include/stdio.h:866,
                 from Src/PoissonRecon.cpp:40:
/usr/include/x86_64-linux-gnu/bits/stdio2.h:38:34: note: ‘__builtin___sprintf_chk’ output between 5 and 1028 bytes into a destination of size 1024
   38 |   return __builtin___sprintf_chk (__s, __USE_FORTIFY_LEVEL - 1,
      |          ~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   39 |       __glibc_objsize (__s), __fmt,
      |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   40 |       __va_arg_pack ());
      |       ~~~~~~~~~~~~~~~~~
Src/PoissonRecon.cpp:375:82: warning: ‘PR_’ directive writing 3 bytes into a region of size between 1 and 1024 [-Wformat-overflow=]
  375 |   if( tempPath[ strlen( tempPath )-1 ]==FileSeparator ) sprintf( tempHeader , "%sPR_" , tempPath );
      |                                                                                  ^~~
In file included from /usr/include/stdio.h:866,
                 from Src/PoissonRecon.cpp:40:
/usr/include/x86_64-linux-gnu/bits/stdio2.h:38:34: note: ‘__builtin___sprintf_chk’ output between 4 and 1027 bytes into a destination of size 1024
   38 |   return __builtin___sprintf_chk (__s, __USE_FORTIFY_LEVEL - 1,
      |          ~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   39 |       __glibc_objsize (__s), __fmt,
      |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   40 |       __va_arg_pack ());
      |       ~~~~~~~~~~~~~~~~~
Src/PoissonRecon.cpp:376:84: warning: ‘PR_’ directive writing 3 bytes into a region of size between 0 and 1023 [-Wformat-overflow=]
  376 |   else                                                  sprintf( tempHeader , "%s%cPR_" , tempPath , FileSeparator );
      |                                                                                    ^~~
In file included from /usr/include/stdio.h:866,
                 from Src/PoissonRecon.cpp:40:
/usr/include/x86_64-linux-gnu/bits/stdio2.h:38:34: note: ‘__builtin___sprintf_chk’ output between 5 and 1028 bytes into a destination of size 1024
   38 |   return __builtin___sprintf_chk (__s, __USE_FORTIFY_LEVEL - 1,
      |          ~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   39 |       __glibc_objsize (__s), __fmt,
      |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   40 |       __va_arg_pack ());
      |       ~~~~~~~~~~~~~~~~~
Src/PoissonRecon.cpp:375:82: warning: ‘PR_’ directive writing 3 bytes into a region of size between 1 and 1024 [-Wformat-overflow=]
  375 |   if( tempPath[ strlen( tempPath )-1 ]==FileSeparator ) sprintf( tempHeader , "%sPR_" , tempPath );
      |                                                                                  ^~~
In file included from /usr/include/stdio.h:866,
                 from Src/PoissonRecon.cpp:40:
/usr/include/x86_64-linux-gnu/bits/stdio2.h:38:34: note: ‘__builtin___sprintf_chk’ output between 4 and 1027 bytes into a destination of size 1024
   38 |   return __builtin___sprintf_chk (__s, __USE_FORTIFY_LEVEL - 1,
      |          ~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   39 |       __glibc_objsize (__s), __fmt,
      |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   40 |       __va_arg_pack ());
      |       ~~~~~~~~~~~~~~~~~
Src/PoissonRecon.cpp:376:84: warning: ‘PR_’ directive writing 3 bytes into a region of size between 0 and 1023 [-Wformat-overflow=]
  376 |   else                                                  sprintf( tempHeader , "%s%cPR_" , tempPath , FileSeparator );
      |                                                                                    ^~~
In file included from /usr/include/stdio.h:866,
                 from Src/PoissonRecon.cpp:40:
/usr/include/x86_64-linux-gnu/bits/stdio2.h:38:34: note: ‘__builtin___sprintf_chk’ output between 5 and 1028 bytes into a destination of size 1024
   38 |   return __builtin___sprintf_chk (__s, __USE_FORTIFY_LEVEL - 1,
      |          ~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   39 |       __glibc_objsize (__s), __fmt,
      |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   40 |       __va_arg_pack ());
      |       ~~~~~~~~~~~~~~~~~
Src/PoissonRecon.cpp:375:82: warning: ‘PR_’ directive writing 3 bytes into a region of size between 1 and 1024 [-Wformat-overflow=]
  375 |   if( tempPath[ strlen( tempPath )-1 ]==FileSeparator ) sprintf( tempHeader , "%sPR_" , tempPath );
      |                                                                                  ^~~
In file included from /usr/include/stdio.h:866,
                 from Src/PoissonRecon.cpp:40:
/usr/include/x86_64-linux-gnu/bits/stdio2.h:38:34: note: ‘__builtin___sprintf_chk’ output between 4 and 1027 bytes into a destination of size 1024
   38 |   return __builtin___sprintf_chk (__s, __USE_FORTIFY_LEVEL - 1,
      |          ~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   39 |       __glibc_objsize (__s), __fmt,
      |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   40 |       __va_arg_pack ());
      |       ~~~~~~~~~~~~~~~~~
Src/PoissonRecon.cpp: In function ‘void Execute(UIntPack<CSignatures ...>, const AuxDataFactory&) [with Real = float; AuxDataFactory = VertexFactory::RGBColorFactory<float>; unsigned int ...FEMSigs = {3, 3, 3}]’:
Src/PoissonRecon.cpp:376:84: warning: ‘PR_’ directive writing 3 bytes into a region of size between 0 and 1023 [-Wformat-overflow=]
  376 |   else                                                  sprintf( tempHeader , "%s%cPR_" , tempPath , FileSeparator );
      |                                                                                    ^~~
In file included from /usr/include/stdio.h:866,
                 from Src/PoissonRecon.cpp:40:
/usr/include/x86_64-linux-gnu/bits/stdio2.h:38:34: note: ‘__builtin___sprintf_chk’ output between 5 and 1028 bytes into a destination of size 1024
   38 |   return __builtin___sprintf_chk (__s, __USE_FORTIFY_LEVEL - 1,
      |          ~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   39 |       __glibc_objsize (__s), __fmt,
      |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   40 |       __va_arg_pack ());
      |       ~~~~~~~~~~~~~~~~~
Src/PoissonRecon.cpp:375:82: warning: ‘PR_’ directive writing 3 bytes into a region of size between 1 and 1024 [-Wformat-overflow=]
  375 |   if( tempPath[ strlen( tempPath )-1 ]==FileSeparator ) sprintf( tempHeader , "%sPR_" , tempPath );
      |                                                                                  ^~~
In file included from /usr/include/stdio.h:866,
                 from Src/PoissonRecon.cpp:40:
/usr/include/x86_64-linux-gnu/bits/stdio2.h:38:34: note: ‘__builtin___sprintf_chk’ output between 4 and 1027 bytes into a destination of size 1024
   38 |   return __builtin___sprintf_chk (__s, __USE_FORTIFY_LEVEL - 1,
      |          ~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   39 |       __glibc_objsize (__s), __fmt,
      |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   40 |       __va_arg_pack ());
      |       ~~~~~~~~~~~~~~~~~
Src/PoissonRecon.cpp:376:84: warning: ‘PR_’ directive writing 3 bytes into a region of size between 0 and 1023 [-Wformat-overflow=]
  376 |   else                                                  sprintf( tempHeader , "%s%cPR_" , tempPath , FileSeparator );
      |                                                                                    ^~~
In file included from /usr/include/stdio.h:866,
                 from Src/PoissonRecon.cpp:40:
/usr/include/x86_64-linux-gnu/bits/stdio2.h:38:34: note: ‘__builtin___sprintf_chk’ output between 5 and 1028 bytes into a destination of size 1024
   38 |   return __builtin___sprintf_chk (__s, __USE_FORTIFY_LEVEL - 1,
      |          ~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   39 |       __glibc_objsize (__s), __fmt,
      |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   40 |       __va_arg_pack ());
      |       ~~~~~~~~~~~~~~~~~
Src/PoissonRecon.cpp:375:82: warning: ‘PR_’ directive writing 3 bytes into a region of size between 1 and 1024 [-Wformat-overflow=]
  375 |   if( tempPath[ strlen( tempPath )-1 ]==FileSeparator ) sprintf( tempHeader , "%sPR_" , tempPath );
      |                                                                                  ^~~
In file included from /usr/include/stdio.h:866,
                 from Src/PoissonRecon.cpp:40:
/usr/include/x86_64-linux-gnu/bits/stdio2.h:38:34: note: ‘__builtin___sprintf_chk’ output between 4 and 1027 bytes into a destination of size 1024
   38 |   return __builtin___sprintf_chk (__s, __USE_FORTIFY_LEVEL - 1,
      |          ~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   39 |       __glibc_objsize (__s), __fmt,
      |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   40 |       __va_arg_pack ());
      |       ~~~~~~~~~~~~~~~~~
Src/PoissonRecon.cpp:376:84: warning: ‘PR_’ directive writing 3 bytes into a region of size between 0 and 1023 [-Wformat-overflow=]
  376 |   else                                                  sprintf( tempHeader , "%s%cPR_" , tempPath , FileSeparator );
      |                                                                                    ^~~
In file included from /usr/include/stdio.h:866,
                 from Src/PoissonRecon.cpp:40:
/usr/include/x86_64-linux-gnu/bits/stdio2.h:38:34: note: ‘__builtin___sprintf_chk’ output between 5 and 1028 bytes into a destination of size 1024
   38 |   return __builtin___sprintf_chk (__s, __USE_FORTIFY_LEVEL - 1,
      |          ~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   39 |       __glibc_objsize (__s), __fmt,
      |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   40 |       __va_arg_pack ());
      |       ~~~~~~~~~~~~~~~~~
Src/PoissonRecon.cpp:375:82: warning: ‘PR_’ directive writing 3 bytes into a region of size between 1 and 1024 [-Wformat-overflow=]
  375 |   if( tempPath[ strlen( tempPath )-1 ]==FileSeparator ) sprintf( tempHeader , "%sPR_" , tempPath );
      |                                                                                  ^~~
In file included from /usr/include/stdio.h:866,
                 from Src/PoissonRecon.cpp:40:
/usr/include/x86_64-linux-gnu/bits/stdio2.h:38:34: note: ‘__builtin___sprintf_chk’ output between 4 and 1027 bytes into a destination of size 1024
   38 |   return __builtin___sprintf_chk (__s, __USE_FORTIFY_LEVEL - 1,
      |          ~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   39 |       __glibc_objsize (__s), __fmt,
      |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   40 |       __va_arg_pack ());
      |       ~~~~~~~~~~~~~~~~~
Src/PoissonRecon.cpp:376:84: warning: ‘PR_’ directive writing 3 bytes into a region of size between 0 and 1023 [-Wformat-overflow=]
  376 |   else                                                  sprintf( tempHeader , "%s%cPR_" , tempPath , FileSeparator );
      |                                                                                    ^~~
In file included from /usr/include/stdio.h:866,
                 from Src/PoissonRecon.cpp:40:
/usr/include/x86_64-linux-gnu/bits/stdio2.h:38:34: note: ‘__builtin___sprintf_chk’ output between 5 and 1028 bytes into a destination of size 1024
   38 |   return __builtin___sprintf_chk (__s, __USE_FORTIFY_LEVEL - 1,
      |          ~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   39 |       __glibc_objsize (__s), __fmt,
      |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   40 |       __va_arg_pack ());
      |       ~~~~~~~~~~~~~~~~~
Src/PoissonRecon.cpp:375:82: warning: ‘PR_’ directive writing 3 bytes into a region of size between 1 and 1024 [-Wformat-overflow=]
  375 |   if( tempPath[ strlen( tempPath )-1 ]==FileSeparator ) sprintf( tempHeader , "%sPR_" , tempPath );
      |                                                                                  ^~~
In file included from /usr/include/stdio.h:866,
                 from Src/PoissonRecon.cpp:40:
/usr/include/x86_64-linux-gnu/bits/stdio2.h:38:34: note: ‘__builtin___sprintf_chk’ output between 4 and 1027 bytes into a destination of size 1024
   38 |   return __builtin___sprintf_chk (__s, __USE_FORTIFY_LEVEL - 1,
      |          ~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   39 |       __glibc_objsize (__s), __fmt,
      |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   40 |       __va_arg_pack ());
      |       ~~~~~~~~~~~~~~~~~
[ 60%] Performing install step for 'poissonrecon'
[ 60%] Completed 'poissonrecon'
[ 60%] Built target poissonrecon
make: *** [Makefile:103: all] Error 2
(odmdev)[bonaime:/code] 12m13s 2 $ 
```

Ok the first Error I have is 
```-- Installing: /home/bonaime/.cmake/packages/gflags/c872b97259e15f7dbf4904fb8eb2bb74
CMake Error at cmake_install.cmake:125 (file):
  file INSTALL cannot copy file
  "/code/SuperBuild/build/gflags/gflags-registry-entry" to
  "/home/bonaime/.cmake/packages/gflags/c872b97259e15f7dbf4904fb8eb2bb74":
  Permission denied.


make[3]: *** [Makefile:105: install] Error 1
/code/SuperBuild/src/lastools/src/geoprojectionconverter.cpp: In member function ‘bool GeoProjectionConverter::set_VerticalCSTypeGeoKey(short int, char*)’:
/code/SuperBuild/src/lastools/src/geoprojectionconverter.cpp:4266:53: warning: format not a string literal and no format arguments [-Wformat-security]
 4266 |           if (description) sprintf(description, name);
      |                                                     ^
make[2]: *** [CMakeFiles/gflags.dir/build.make:93: gflags/stamp/gflags-install] Error 2
make[1]: *** [CMakeFiles/Makefile2:499: CMakeFiles/gflags.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 30%] Building CXX object libs/mve/CMakeFiles/mve.dir/mesh_io_ply.cc.o
```

And I have those warnings
````
Installing collected packages: vmem, Shapely, repoze.lru, pytz, Pysolar, ODMExifRead, mpmath, flatbuffers, xmltodict, wrapt, urllib3, typing
-extensions, threadpoolctl, sympy, soupsieve, six, PyYAML, pyparsing, protobuf, Pillow, piexif, packaging, numpy, mypy-extensions, munch, lx
ml, kiwisolver, joblib, idna, humanfriendly, decorator, cycler, cloudpickle, click, charset-normalizer, certifi, attrs, affine, typing-inspe
ct, trimesh, tifffile, snuggs, scipy, requests, Pywavelets, python-dateutil, pyproj, networkx, marshmallow, imageio, fpdf2, edt, deprecated,
 coloredlogs, codem, cligj, click-plugins, beautifulsoup4, appsettings, scikit-learn, scikit-image, requests-toolbelt, rasterio, pandas, onn
xruntime, matplotlib, Fiona, dataclasses-json, pyodm, pygltflib
  WARNING: The script isympy is installed in '/home/bonaime/.local/bin' which is not on PATH.
  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.
  WARNING: The scripts f2py, f2py3 and f2py3.9 are installed in '/home/bonaime/.local/bin' which is not on PATH.
  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.
  WARNING: The script humanfriendly is installed in '/home/bonaime/.local/bin' which is not on PATH.
  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.
  WARNING: The script normalizer is installed in '/home/bonaime/.local/bin' which is not on PATH.
  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.
  WARNING: The scripts lsm2bin, tiff2fsspec, tiffcomment and tifffile are installed in '/home/bonaime/.local/bin' which is not on PATH.
  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.
  WARNING: The script pyproj is installed in '/home/bonaime/.local/bin' which is not on PATH.
`````

--- a/SuperBuild/CMakeLists.txt
+++ b/SuperBuild/CMakeLists.txt
@@ -142,7 +142,7 @@ SETUP_EXTERNAL_PROJECT(OpenCV ${ODM_OpenCV_Version} ${ODM_BUILD_OpenCV})
 # ---------------------------------------------------------------------------------------------
 # Google Flags library (GFlags)
 #
-set(ODM_GFlags_Version 2.1.2)
+set(ODM_GFlags_Version 2.2.2)
 option(ODM_BUILD_GFlags "Force to build GFlags library" OFF)
 
 SETUP_EXTERNAL_PROJECT(GFlags ${ODM_GFlags_Version} ${ODM_BUILD_GFlags})

